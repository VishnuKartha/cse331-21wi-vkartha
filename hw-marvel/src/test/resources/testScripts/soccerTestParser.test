# Test the parser on a the soccer database that contains several nodes
# with two edges each.

LoadGraph soccer soccer.tsv
ListChildren soccer Neymar
