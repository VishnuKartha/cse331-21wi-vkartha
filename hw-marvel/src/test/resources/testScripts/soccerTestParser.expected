# Test the parser on a the soccer database that contains several nodes
# with two edges each.

loaded graph soccer
the children of Neymar in soccer are: Leo Messi(Barcelona) Mbappe(PSG)